tests/test_worker.py

```diff
@@ -7,7 +7,7 @@
 from . import settings
 
 
-@sub(topic='some-cool-topic')
+@sub(topic='some-cool-topic', prefix='rele')
```

Comment:
Please remove this stub. Is duplicated and not used in worker's test context.

Reply:
Ok, now I see it's used not only here but in `test_subscription.py` as well.

Please move it to a `conftest.py` file in an upper level.

Reply:
Because is not in other test modules, there is no reason to create `conftest.py`. Alternate approach should be to move it to the test function.